### DISCUSSION TITLE
Starting Tracks from boot

### DISCUSSION THREAD

**Discussion by DPBiff** [TracksApp/tracks#3029] [Q&A]
I have Tracks up and running on a Linux server.  I start it successfully from a terminal. But when the terminal window closes so does the app.  I tried starting with a cron job but no luck what's the process?

**Comment by ZeiP** [comment]
That depends on your operating system etc. I would probably configure a Systemctl job of it. Some instructions are available for example at https://www.reddit.com/r/ruby/comments/6ek4og/run_ruby_app_as_a_linux_service/ and https://medium.com/the-blockchain-artist/running-ruby-on-rails-5-application-as-systemctl-6fbe961e72dc .
